src/ApiApprover/PublicApiApprover.cs

```diff
@@ -1,12 +1,15 @@
-﻿using System.IO;
+﻿using System;
+using System.IO;
 using System.Reflection;
 using ApprovalTests;
 using ApprovalTests.Namers;
 
 namespace ApiApprover
 {
+    [Obsolete("PublicApiApprover will be removed in the next major version. Either use the `ApiGenerator` directly or copy https://github.com/JakeGinnivan/ApiApprover/blob/master/src/ApiApprover/PublicApiApprover.cs into your repository if you plan to continue to use ApprovalTests.", false)]
```

Comment:
@bording spell checking would be highly appreciated ;)

Reply:
This seems a bit strange to me, because you'll actually just stop publishing the ApiApprover package, right? The PublicApiGenerator package would be the only one still being published?

If I'm understanding the plan correctly, then these feel misplaced since you'd actually want to tell people to migrate to a new package instead.

Reply:
Changed the message